src/webgpu/shader/execution/expression/call/builtin/texture_utils.ts

```diff
@@ -191,7 +191,10 @@ export function skipIfTextureFormatNotSupportedNotAvailableOrNotFilterable(
 }
 
 const builtinNeedsMipGradientValues = (builtin: TextureBuiltin) =>
```

Comment:
If I'm understanding this variable name correctly, shouldn't `textureSampleGrad`, `textureSampleLevel`, and `textureSampleCompareLevel` also be listed?

LGTM regardless though.

Reply:
renamed to something more appropriate